src/test/java/utilities/Util.java

```diff
@@ -13,4 +17,17 @@ public static void wait(int seconds) {
 		}
 	}
 
+	/*
+	 * Select Drop-down option
+	 * @author Wenji Zou
+	 */
+	public static void dropDownSelect(WebDriver driver, String dorpDownXpath, String optionValue) {
+		Select dropDown = new Select(driver.findElement(By.xpath(dorpDownXpath)));
```

Comment:
1, Instead of xpath as a parameter, pass in a `By` object, makes it more reusable
2, make sure you format the file